### ISSUE TITLE
Unable to install ffmpeg from source

### ISSUE THREAD

**Issue by rmrfslashbin** [skycocker/chromebrew#1618]
## Description
ffmpeg fails to build when installing from source.

If you executed a specific command:
```
chronos@localhost ~ $ crew install -s ffmpeg
```

Output:
```
...

License: nonfree and unredistributable
Creating configuration files ...
awk: run time error: redirection not allowed in sandbox mode
        FILENAME="" FNR=0 NR=0
awk: run time error: redirection not allowed in sandbox mode
        FILENAME="" FNR=0 NR=0
awk: run time error: redirection not allowed in sandbox mode
        FILENAME="" FNR=0 NR=0
awk: run time error: redirection not allowed in sandbox mode
        FILENAME="" FNR=0 NR=0
Preconfiguring package...
make: *** No rule to make target 'install-libs-yes', needed by 'install-libs'.  Stop.
ffmpeg failed to install: `make DESTDIR=/usr/local/tmp/crew/dest install` exited with 2
chronos@localhost ~ $ 
```

## Versions and system information
chromebrew: `0.4.3`

ChromeOS: `chronos@localhost ~ $ cat /etc/lsb-release 
CHROMEOS_ARC_ANDROID_SDK_VERSION=25
CHROMEOS_ARC_VERSION=4510202
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={01906EA2-3EB2-41F1-8F62-F0B7120EFD2E}
CHROMEOS_RELEASE_BOARD=eve-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=22
CHROMEOS_RELEASE_BUILDER_PATH=eve-release/R64-10176.22.0
CHROMEOS_RELEASE_BUILD_NUMBER=10176
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=64
CHROMEOS_RELEASE_DESCRIPTION=10176.22.0 (Official Build) beta-channel eve 
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=beta-channel
CHROMEOS_RELEASE_VERSION=10176.22.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=10176.22.0`

Model: Pixelbook

Architecture: `x86_64`

**Comment by uberhacker** [comment]
Reopened this issue since version 3.4.2 is having the same problem now.

**Comment by uberhacker** [comment]
Fixed in #2858.
